opslevel/resource_opslevel_check_service_property.go

```diff
@@ -104,6 +109,12 @@ func (r *CheckServicePropertyResource) Schema(ctx context.Context, req resource.
 		MarkdownDescription: "Check Service Property Resource",
 
 		Attributes: CheckBaseAttributes(map[string]schema.Attribute{
+			"component_type": schema.StringAttribute{
+				Description: "The Component Type that a custom property belongs to. Defaults to Service properties if not provided.",
+				Optional:    true,
+				Computed:    true,
+				Default:     stringdefault.StaticString("service"),
+			},
```

Comment:
We're making changes to apply filters/checks to properties based solely on the alias (allowing it to scope _all_ properties with a given alias) so this might have resolved a different way. However, we'll continue to use the component_type to scope the property alias down optionally too - so I definitely makes sense to continue with this change.